code/modules/mob/living/carbon/human/human.dm

```diff
@@ -269,13 +269,14 @@
 					return
 				to_chat(usr, "<b>Name:</b> [R.fields["name"]] <b>Criminal Status:</b> [R.fields["criminal"]]")
 				for(var/datum/data/crime/c in R.fields["crim"])
-					to_chat(usr, "<b>Crime:</b> [c.crimeName]")
-					if (c.crimeDetails)
-						to_chat(usr, "<b>Details:</b> [c.crimeDetails]")
-					else
-						to_chat(usr, "<b>Details:</b> <A href='?src=[REF(src)];hud=s;add_details=1;cdataid=[c.dataId]'>\[Add details]</A>")
-					to_chat(usr, "Added by [c.author] at [c.time]")
-					to_chat(usr, "----------")
+					if(c.crimeStatus != "Served")
```

Comment:
Why shouldn't we show served crimes?

Reply:
The idea is that the sechuds should display the info that's immediately important in a concise manner, so we only show the currently active crimes. Since you can't check crimes on the tablet anymore the feature might be missed, but if it gets people to communicate on the radio more I'm all for it.